sycl/include/CL/sycl/handler.hpp

```diff
@@ -382,8 +382,11 @@ class handler {
           std::move(MSharedPtrStorage), std::move(MRequirements),
           std::move(MEvents)));
       break;
+    case detail::CG::NONE:
+      throw runtime_error("Command group submitted without a kernel or a "
+                          "explicit memory operation.");
     default:
-      throw runtime_error("Unhandled type of command group");
+      throw runtime_error("Unhanded type of command group");
```

Comment:
"Unhandled" is more correct than "Unhanded" here.

Reply:
Fixed.